docs/getting-started/list-filtering/django.rst

```diff
@@ -113,15 +113,27 @@ For example, the above policy has the following rule:
     :language: polar
     :lines: 4-6
 
-When determining which ``Post`` objects ``User(id=1)`` is authorized to see,
+When determining which ``Post`` objects ``User(id=2)`` is authorized to see,
 the ``django-oso`` adapter converts the constraints on Post expressed in this
 rule into a Django ``Q`` filter::
 
-    (AND: ('access_level', 'private'), ('creator__pk', 1))
+    (AND: ('access_level', 'private'), ('creator__pk', 2))
 
-This filter then scopes down the QuerySet to only include authorized objects.
-Translating from policy to filter to authorized data makes the policy an
-effective abstraction for expressing authorization logic over collections.
+When composed with filters generated from the other rules, the QuerySet is
```

Comment:
Might be nice to link to code enabling the SQL logging.

Reply:
Talked about this offline -- it's fine w/o.